.circleci/config.yml

```diff
@@ -53,13 +53,30 @@ save-gems-cache: &save-gems-cache
   paths:
     - vendor/bundle
 
-update-fastlane: &update-fastlane
+update-fastlane-ios: &update-fastlane-ios
   name: Update Fastlane
   command: |
     echo "ruby-2.6.4" > ~/.ruby-version
-    bundle install
+    bundle install && bundle update
   working_directory: ios
 
+update-fastlane-android: &update-fastlane-android
+  name: Update Fastlane
+  command: |
+    echo "ruby-2.6.4" > ~/.ruby-version
+    bundle install && bundle update
+  working_directory: android
```

Comment:
Why did you reverted the command using param?

Reply:
I tried the way suggested by Diego and there seems some pb in it so he suggested to revert back